src/virt-install

```diff
@@ -51,6 +51,14 @@ def get_libvirt_smp_arg():
     See e.g. https://www.redhat.com/archives/libvirt-users/2017-June/msg00025.html
     We want to match the number of host physical cores.
     """
+
+    # When running under K8S, the scehduling may be much more restrictive than
+    # physical core count, and the code below will assing that count, leading
+    # to a VM hang. (e.g. 64 vcpu's to a 1 cpu assignment...)
+    p1c = open("/proc/1/cgroup")
+    if re.search(r'.*kubepods.*', p1c.read()):
+        return '--vcpus=sockets=1,cores=1,threads=1'
```

Comment:
Humm...I swear we tested this though.  When I e.g. rsh into a MCD pod and type `nproc` I get `2` which matches the host.

You're saying this code is returning `64` today?

Reply:
Right, I saw this in OpenShift where I had a resource limit of 2CPU's. Looking at the logs, it was clearly getting 64.